### PR TITLE
Update roadmap for Q2 2022

### PR DESCRIPTION
**Pull request by crodriguezvega**

### CODE REVIEW

File: meta/ROADMAP.md [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
+- The [connection](https://github.com/cosmos/ibc/pull/621) and [channel](https://github.com/cosmos/ibc/pull/677) upgradability specs have been merged, but they need some small fixes. The spec team will also help with the planning of the implementation of channel upgradability in [ibc-go](https://github.com/cosmos/ibc-go).
+- Finish writing the spec for [ordered channels that support timeouts](https://github.com/cosmos/ibc/pull/636).
+- Start writing the spec to support state trees without absence proofs.
+- The implementation of [ICS29](https://github.com/cosmos/ibc/tree/master/spec/app/ics-029-fee-payment) in ibc-go will be finished in Q2 and the spec might need some updates to reflect the latest status.
+- Finish ICS28 (Cross-chain validation) spec.
```
Comment (mpoke) [inline meta/ROADMAP.md:16]:
```suggestion
- Finish [ICS28](https://github.com/cosmos/ibc/pull/666) (Cross-chain validation) spec.
```

**Review by mpoke — APPROVED** [review]
LGTM

**Review by josef-widder — APPROVED** [review]
Looks good. Thanks!

File: meta/ROADMAP.md [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
-- Work on the connection upgradability and channel upgradability specs.
-- Work on spec for ordered channels that are allowed to timeout.
-- Work on spec to support timeouts without proof of absence.
-- Finalize a complete draft of ICS28 (Cross-chain validation) that is aligned with the implementation of Interchain Security V1.
-- Review [IRISnet](https://www.irisnet.org)'s [ICS721](https://github.com/cosmos/ibc/pull/615) specification proposal for NFT transfers.
\ No newline at end of file
+- Work on general readability improvements and inconsisntency fixes in some of the specs ([ICS02](https://github.com/cosmos/ibc/blob/carlos/update-roadmap/spec/core/ics-002-client-semantics/README.md), [ICS06](https://github.com/cosmos/ibc/blob/carlos/update-roadmap/spec/client/ics-006-solo-machine-client/README.md), [ICS07](https://github.com/cosmos/ibc/blob/carlos/update-roadmap/spec/client/ics-007-tendermint-client/README.md)). This is a first step on the long-term plan to make the specs easier to understand to qualified developers.
```
Comment (mpoke) [inline meta/ROADMAP.md:11]:
```suggestion
- Work on general readability improvements and inconsintency fixes in some of the specs ([ICS02](https://github.com/cosmos/ibc/blob/master/spec/core/ics-002-client-semantics/README.md), [ICS06](https://github.com/cosmos/ibc/blob/master/spec/client/ics-006-solo-machine-client/README.md), [ICS07](https://github.com/cosmos/ibc/blob/master/spec/client/ics-007-tendermint-client/README.md)). This is a first step on the long-term plan to make the specs easier to understand to qualified developers.
```

Comment (AdityaSripal) [inline meta/ROADMAP.md:11]:
```suggestion
- Work on general readability improvements and inconsistency fixes in some of the specs ([ICS02](https://github.com/cosmos/ibc/blob/carlos/update-roadmap/spec/core/ics-002-client-semantics/README.md), [ICS06](https://github.com/cosmos/ibc/blob/carlos/update-roadmap/spec/client/ics-006-solo-machine-client/README.md), [ICS07](https://github.com/cosmos/ibc/blob/carlos/update-roadmap/spec/client/ics-007-tendermint-client/README.md)). This is a first step on the long-term plan to make the specs easier to understand to qualified developers.
```

**Review by AdityaSripal — APPROVED** [review]

**Pull request merged**